woocommerce/woocommerce
# Add a new icon style to the Customer Account block

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/48793

This PR introduces a new icon style option to the Customer Account block.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new page or post.
2. Insert the `Customer Account` block.
3. Check there are 3 styles of icons to choose from on the sidebar settings:
<img width="273" alt="Screenshot 2024-07-01 at 13 49 49" src="https://github.com/woocommerce/woocommerce/assets/186112/e1350493-33a5-4ceb-a36f-02b26b8f8089">

4. Choose the first one (it's the new style).
5. Check the block in the editor updates accordingly.
6. Save and check it displays as well in the front end.
7. Do some extra testing by choosing other `Icon options` and `Display style` and check they show correctly on the editor and the front end.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Add a new icon style to the "Customer Account" block.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
